Realm.PCL/Schema/RealmSchemaPCL.cs

```diff
@@ -22,21 +22,43 @@
 
 namespace Realms
 {
+    /// <summary>
+    /// Describes the complete set of classes which may be stored in a Realm, either from assembly declarations or, dynamically, by evaluating a Realm from disk.
+    /// </summary>
+    /// <remarks>
+    /// By default this will be all the RealmObjects in all your assemblies unless you restrict with RealmConfiguration.ObjectClasses. 
+    /// Just because a given class <em>may</em> be stored in a Realm doesn't imply much overhead. There will be a small amount of metadata 
```

Comment:
This doesn't seem like the right place for this bit of information to me. We could add it to the FAQ if necessary, or in some general documentation somewhere.